tiatoolbox/annotation/storage.py

```diff
@@ -1003,6 +1004,14 @@ def __init__(
         ):
             raise Exception("RTREE and JSON1 sqlite3 compile options are required.")
 
+        # Check that math functions are enabled
+        if "ENABLE_MATH_FUNCTIONS" not in compile_options:
+            logger.warning(
+                "SQLite math functions are not enabled."
+                " This may cause problems with some queries."
+                " For example, floor division (//) will not work."
+            )
+
```

Comment:
It would be more helpful if you put the cases that could use // underneath within the doctring, and then add the URL to readthedoc in the warning here for fixing direction.

Reply:
Just added a documentation build to check this. Currently I think it is only `//` that will be affected.

Reply:
See https://tia-toolbox.readthedocs.io/en/bug-sqlite-floor/_autosummary/tiatoolbox.annotation.dsl.html

Reply:
Looking good